src/config/rounds/round1.spec.tsx

```diff
@@ -58,55 +52,44 @@ describe('round 1', () => {
         const game = getGame();
 
         game.selectAction('ACTION_PROTECTED_FROM_OUTSIDE_DISTRACTION');
+        expect(game.state.currentRound.userStoryChance).toEqual(40);
 
-        // TODO Hannes - very very confused as to how/what this is checking
-        // it would help if this was extracted to a testHelper so it could be used to test multiple actions
-        addRolls(Array(game.state.currentRound.capacity.available).fill(0.39));
-        game.nextRound();
-        expect(game.state.result.storiesCompleted).toEqual(9);
-
-        expect(game.state.currentRound.capacity.available).toEqual(10);
-
-        addRolls(Array(5).fill(0.39));
-        addRolls(Array(5 + /* gremlin */ 2).fill(0.41));
-        game.nextRound();
-        expect(game.state.result.storiesCompleted).toEqual(14);
-
-        // proving it had no effect on capacity
-        // TODO - Hannes help - we can't call here - becaue it uses the wrong "random effect" testFutureCapacities(game, [10, 10, 10, 10]);
+        times(5, () => {
+          game.nextRound();
+          expect(game.state.currentRound.capacity.total).toEqual(10);
+          expect(game.state.currentRound.userStoryChance).toEqual(40);
+        });
```

Comment:
I like `testFutureCapacities` and at first i added a `testFutureUserStoryChance` with a similar signature. 
The problem was that both internally called `game.nextRound()`

```ts
testFutureCapacities(game, [10, 10]); // tests round 2 + 3
testFutureUserStoryChance(game, [40, 40]); // tests round 4 + 5
```

we could have merged that to s.th. like
```ts
testFutureState(game, [{ capacity: 10, storyChance: 40 }, { capacity: 10, storyChance: 40 }]);
```

but i feel that would have blown up the test-helper so I went with an iteration in the test itself

Reply:
FWIW Your idea of `testFutureState `is inline with my mental model.

`testFutureCapacities ` simply came about because I could see masses of code duplication coming up as I ad more actions.